src/api/common/schemas/quality.json

```diff
@@ -1,8 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "quality",
-  "description": "Ratio for incoming/outgoing transit fees represented in billionths. (For example, a value of 500 million represents a 0.5:1 ratio.) As a special case, 0 is treated as a 1:1 ratio.",
-  "type": "integer",
-  "minimum": 0,
-  "maximum": 1000000000
+  "description": "Ratio for incoming/outgoing transit fees. As a special case, 0 is treated as a 1:1 ratio.",
+  "type": "number",
+  "minimum": 0
```

Comment:
The transferRate schema has a maximum defined by the UInt32 upper bound. Why not quality also?  As well as lower/upper bounds I guess it should also check if there's more than 9 decimal places? What was the reason to use a float instead of a string, consistent with amount@value, and trustline@limit, etc?